README.md

```diff
@@ -1,2 +1,140 @@
-# Capstone
-Urban Design &amp; Landscape Expo
+# My-Professional-Portofolio
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+  <br/>
+
+  <h3><b>Urban Design & Landscape Expo 2023</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 My Portofolio <a name="about-project"></a>
+
+This project has been created for Big 5. Which is the largest and most influential event for the construction industry with its global hub in Dubai acting as the gateway between East and West.
+
+Our mission is to unite the global construction community and provide leading-edge innovation, knowledge, and business opportunities for industry stakeholders to connect, grow, and succeed.
+
+We believe in accelerating progress in the construction industry to support economic growth and promote a healthier and more sustainable built environment around the world.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML.
+- CSS.
+- JavaScript
+- LINTERS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Big5</a></li>
+    <li><a href="https://reactjs.org/">Urban Design & Landscape Expo</a></li>
+    <li><a href="https://reactjs.org/"></a>Dubai Expo 2020</li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Added html.index file.
+- Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web Browsers
+- Code Editor.
+- Git -smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command.
+
+### Install
+
+Install this project with:
+Run this command:
+
+- cd my-project
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+OPen index.html using live server.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Qaisar Abbas**
```

Comment:
- [ ] please kindly ensure to deploy your project. makes use of GitHub Pages to deploy the web pages.
- ensure the project is deployed properly, all pages, links, and images work as expected.
- Lastly include a link to the deployed project in the README.